docs/data/material/pages.ts

```diff
@@ -310,6 +310,22 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/material-ui/design-resources',
+    title: 'Design resources',
+    children: [
+      {
+        pathname: '/material-ui/design-resources/mui-connect',
+        title: 'MUI Connect plugin',
+        newFeature: true,
+      },
+      {
+        pathname: '/material-ui/design-resources/material-ui-for-figma',
+        title: 'Material UI for Figma',
+        newFeature: true,
+      },
```

Comment:
I would reverse the order of these two links; Designers can't use the MUI Connect plugin without first using Material UI for Figma.

Reply:
Good call, makes sense 👍